$OpenBSD: patch-exempi_exempi_cpp,v 1.1 2018/04/10 16:53:12 jasper Exp $

error: cannot initialize a variable of type 'pthread_key_t' (aka 'int') with an rvalue of type 'nullptr_t'

Index: exempi/exempi.cpp
--- exempi/exempi.cpp.orig
+++ exempi/exempi.cpp
@@ -65,7 +65,7 @@ static void set_error(int err)
 #include <pthread.h>
 
 /* Portable thread local storage using pthreads */
-static pthread_key_t key = NULL;
+static pthread_key_t key;
 static pthread_once_t key_once = PTHREAD_ONCE_INIT;
 
 /* Destructor called when a thread exits - ensure to delete allocated int pointer */ 
